caveclient/tools/testing.py

```diff
@@ -219,6 +219,16 @@ def version_url(
     return version_endpoint.format_map(mapping)
 
 
+def versions_url(
```

Comment:
I'm confused this is a function with no implementation?

Reply:
That's leftover from when I thought I needed a variant on version_url. Thanks for pointing it out. I've removed it.